inc/core/deprecated/deprecated-filters.php

```diff
@@ -85,3 +85,96 @@ function astra_apply_filters_deprecated( $tag, $args, $version, $replacement = f
 		}
 	}
 }
+
+// Deprecating ast_footer_bar_display filter.
+add_filter( 'astra_footer_bar_display', 'astra_deprecated_ast_footer_bar_display_filter', 10, 1 );
+
+/**
+ * Display footer builder markup.
+ *
+ * @since x.x.x
+ * @param boolean $display_footer true | false.
+ * @return boolean true for enabled | false for disable.
+ */
+function astra_deprecated_ast_footer_bar_display_filter( $display_footer ) {
+	return astra_apply_filters_deprecated( 'ast_footer_bar_display', array( $display_footer ), '3.7.4', 'astra_footer_bar_display', '' );
+}
+
+// Deprecating ast_main_header_display filter.
+add_filter( 'astra_main_header_display', 'astra_deprecated_ast_main_header_display_filter', 10, 1 );
+
+/**
+ * Display header builder markup.
+ *
+ * @since x.x.x
+ * @param boolean $display_header true | false.
+ * @return boolean true for enabled | false for disable.
+ */
+function astra_deprecated_ast_main_header_display_filter( $display_header ) {
+
+	return astra_apply_filters_deprecated( 'ast_main_header_display', array( $display_header ), '3.7.4', 'astra_main_header_display', '' );
+}
+
+// Deprecating secondary_submenu_border_class filter.
+add_filter( 'astra_secondary_submenu_border_class', 'astra_deprecated_secondary_submenu_border_class_filter', 10, 1 );
+
+/**
+ * Border class to secondary submenu
+ *
+ * @since x.x.x
+ * @param string $class_selector custom class assigned to secondary submenu.
+ * @return string  $class_selector updated class selector.
+ */
+function astra_deprecated_secondary_submenu_border_class_filter( $class_selector ) {
+
+	$class_selector = astra_apply_filters_deprecated( 'secondary_submenu_border_class', array( $class_selector ), '3.7.4', 'astra_secondary_submenu_border_class', '' );
+
+	return $class_selector;
+}
+
+// Deprecating gtn_image_group_css_comp filter.
+add_filter( 'astra_gutenberg_image_group_style_support', 'astra_deprecated_gtn_image_group_css_comp_filter', 10, 1 );
+
+/**
+ * Image, group compatibility support released in v2.4.4.
+ *
+ * @since x.x.x
+ * @param boolean $block_support true | false.
+ * @return boolean true for enabled | false for disable.
+ */
+function astra_deprecated_gtn_image_group_css_comp_filter( $block_support ) {
+
+	return astra_apply_filters_deprecated( 'gtn_image_group_css_comp', array( $block_support ), '3.7.4', 'astra_gutenberg_image_group_style_support', '' );
+}
+
+// Deprecating ast_footer_sml_layout filter.
+add_filter( 'astra_footer_sml_layout', 'astra_deprecated_ast_footer_sml_layout_filter', 10, 1 );
+
+/**
+ * Footer bar meta setting option.
+ *
+ * @since x.x.x
+ * @param boolean $display_footer_bar true | false.
+ * @return boolean true for enabled | false for disable.
+ */
+function astra_deprecated_ast_footer_sml_layout_filter( $display_footer_bar ) {
+
+	return astra_apply_filters_deprecated( 'ast_footer_sml_layout', array( $display_footer_bar ), '3.7.4', 'astra_footer_sml_layout', '' );
```

Comment:
```suggestion
	return astra_apply_filters_deprecated( 'ast_footer_sml_layout', array( $display_footer_bar ), 'x.x.x', 'astra_footer_sml_layout', '' );
```
@imnavanath Lets keep this `x.x.x`

Reply:
Done